pages/regions.js

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, H6, Icon } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const cta = router.query.redirect;
+
+  return (
+    <section className="regions">
+      {data.regions.map((region) => {
+        return (
+          <div key={nanoid()} region={region.name} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Link href={cta === 'login' ? region.url : region.altUrl}>
+                <a>
+                  <Icon name={region.icon} subStyle={region.color} mainStyle="fill" />
+                </a>
+              </Link>
+            </div>
+            <div className="regions__title">
+              <H6>
+                {region.title}
+              </H6>
+            </div>
+            <div className="regions__text">
+              <Link href={cta === 'login' ? region.url : region.altUrl}>
+                <a>
+                  <H6>
```

Comment:
Again this should be using the `<P>` component instead. You can achieve the proper spacing and alignment by adding a className of `i` to the parent `<a>` element. Just wrap region.text in a span. You can see how this works from the styling here:

https://github.com/jambonz/next-static-site/blob/main/src/styles/_typography.scss#L13-L25

There is an example of this on the Pricing page under Additional Services.